### PR TITLE
Added picture to `readme.md` showing where to find access token from OpenShift homepage

### PR DESCRIPTION
**Pull request by yemi-kelani**
Related to enhancement #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ Single-click actions to Create, Re-deploy, and Delete your operator in OpenShift
 
 Configure your OpenShift server URL, and select your OpenShift Project directly from your VS Code editor
 
+![OpenShift URL and Access Token](./resources/docs/media/oc-get-access-token.png)
+
 ![OpenShift configuration](./resources/docs/media/oc-sdk-openshift-connection.gif)
```
Comment (MariandreaMV) [inline README.md:31]: Wouldn’t be better to re-create this gif but adding the step of where to get the required inputs? just to keep consistency with the other media resources used in this readme file.

Comment (yemi-kelani) [reply README.md:31]: Yes thats a better idea. I can make that change.

Comment (yemi-kelani) [reply README.md:31]: Yes thats a better idea. I can make that change.

**Review by freemanlatrell — APPROVED** [review]

**Review by freemanlatrell — APPROVED** [review]

**Comment by freemanlatrell** [comment]
PR builds not required since this is only a doc update. Merging

**Pull request merged**